libhoney.go

```diff
@@ -572,8 +592,9 @@ func (f *fieldHolder) Fields() map[string]interface{} {
 // fields are specified in neither Config nor the Event, Send will return an
 // error.  Required fields are APIHost, WriteKey, and Dataset. Values specified
 // in an Event override Config.
-func (e *Event) Send() error {
+func (e *Event) Send() (err error) {
```

Comment:
?

Reply:
umm.  I'll second that. ??  
removed.

Reply:
oh, I think I originally put the deferred log line in SendPresampled here, then realized it was better there, and failed to remove the named return value.